src/widgets/_buttons.scss

```diff
@@ -21,7 +21,11 @@ button {
     &.raised.image-button,
     & {
         @extend %outset-background;
-        background-color: bg-color(2);
+        @if $color-scheme == "light" {
+          background-color: bg-color(2);
+        } @else if $color-scheme == "dark" {
+          background-color: mix($BLACK_300, $BLACK_500, $weight: 70%);
+        }
```

Comment:
This is going to make it more difficult to keep things consistent. Instead of doing this, you should change `bg-color(2)` to return this value for the dark style

Reply:
If I do that, then everything at level 2 of bg-color will be as raised as the buttons, defeating some of the purpose of the PR. What should I do then?